django/test/signals.py

```diff
@@ -176,3 +176,16 @@ def auth_password_validators_changed(**kwargs):
     if kwargs['setting'] == 'AUTH_PASSWORD_VALIDATORS':
         from django.contrib.auth.password_validation import get_default_password_validators
         get_default_password_validators.cache_clear()
+
+
+@receiver(setting_changed)
+def localize_settings_changed(**kwargs):
+    localize_settings = {
```

Comment:
Maybe there's a benefit to using `utils.formats.FORMAT_SETTINGS` and adding `USE_THOUSAND_SEPARATOR` to it? For simplicity, even adding `USE_THOUSAND_SEPARATOR` to that list would likely be fine.